test/phoenix_live_view/html_engine_test.exs

```diff
@@ -1317,7 +1484,12 @@ defmodule Phoenix.LiveView.HTMLEngineTest do
 
   describe "html validations" do
     test "phx-update attr requires an unique ID" do
-      message = ~r/.exs:1:(1:)? attribute \"phx-update\" requires the \"id\" attribute to be set/
+      message = """
+      test/phoenix_live_view/html_engine_test.exs:1:1: attribute \"phx-update\" requires the \"id\" attribute to be set
+        |
+      1 | <div phx-update=\"ignore\">
+        |  ^\
```

Comment:
This one is missing ID. Do you think the arrow is correct?

Reply:
This is fine but there are others that point to `<` and maybe we should make them consistent?